### DISCUSSION TITLE
可以加个替换标记{Current_mc_version}是mc当前版本

### DISCUSSION THREAD

**Discussion by WSXYT** [Meloong-Git/PCL#3875] [提问] [answered]
可以加个替换标记{Current_mc_version}是mc当前版本

**Comment by allMagicNB** [comment, marked as answer]
你要不先看看替换标记有啥？

**Reply by WSXYT** [reply]
可是问了一下龙猫说没有这个功能qwq
如果有了话能发一下吗qwq

**Reply by MoYuan-CN** [reply]
![image](https://github.com/Hex-Dragon/PCL2/assets/109946426/0b50007c-7b25-4356-9ff2-1ceeefb08f1d)
<img src="https://github.com/Hex-Dragon/PCL2/assets/109946426/e080042c-86b7-465d-90af-716d46503151" width="120">

**Reply by WSXYT** [reply]
对不起眼瞎了0.0

**Reply by WSXYT** [reply]
![image](https://github.com/Hex-Dragon/PCL2/assets/102407247/196d05b1-b8f2-4c9a-a7e6-8f9c3b9ba199)
不过我用了就报错这个qwq
望大佬可以教一下qwq

**Reply by allMagicNB** [reply]
自定义主页不支持替换标记……我没想到用途在哪（

**Reply by WSXYT** [reply]
一键启动对应的服务器之类的，难到不方便吗（

**Reply by MoYuan-CN** [reply]
> 一键启动对应的服务器之类的，难到不方便吗（

EventData 使用 `版本名|服务器ip`

**Reply by WSXYT** [reply]
问题是我想要启动当前选择的版本（

**Reply by MoYuan-CN** [reply]
当前选择的版……本？
现有的内容不足以支持这个功能
如果你需要的话可以交一个新的反馈
